lib/index.cjs.ts

```diff
@@ -1,6 +1,6 @@
 import jwtDecode, { InvalidTokenError } from "./index";
 
-const wrapper = jwtDecode;
+const wrapper = jwtDecode as any;
 wrapper.default = jwtDecode;
 wrapper.InvalidTokenError = InvalidTokenError;
 export default wrapper;
```

Comment:
Would this work? 🤔  no idea just wondering

```
export default {
  default: jwtDecode,
  InvalidTokenError,
};
```

Reply:
From the looks of it, it doesn't end up to be the same so I'll keep it like this.